priam/src/main/java/com/netflix/priam/config/CassandraConfiguration.java

```diff
@@ -29,6 +29,9 @@
     private String cassHome;
 
     @JsonProperty
+    private String yamlLocation;
```

Comment:
@JsonIgnore seems to disable serialization/deserialization altogether, which removes the ability to override this value in situations where we need to (such as priam_local.yaml). Leaving this line as-is and adding the Objects.firstNonNull() logic you mention below seems to have the desired effect: using the YAML-specified "yamlLocation" when it exists, using the default value when it doesn't.